yarn.lock

```diff
@@ -1569,6 +1577,14 @@ amdefine@>=0.0.4:
   resolved "https://registry.yarnpkg.com/amdefine/-/amdefine-1.0.1.tgz#4a5282ac164729e93619bcfd3ad151f817ce91f5"
   integrity sha1-SlKCrBZHKek2Gbz9OtFR+BfOkfU=
 
+angular-tree-component@^8.5.6:
```

Comment:
Has the yarn.lock file been deleted and rebuild in this PR? Why are there so many changes to the lock file?